pytest_mypy_plugins/tests/test_configs/test_join_toml_configs.py

```diff
@@ -54,7 +61,7 @@ def factory(filename: Optional[str], expected: str) -> None:
 def test_join_existing_config(
     execution_path: Path, assert_file_contents: _AssertFileContents, additional_config: str
 ) -> None:
-    filepath = join_toml_configs(_PYPROJECT1, additional_config, execution_path)
+    filepath = join_toml_configs(_MYPY_PLUGINS_CONFIG2, _PYPROJECT1, additional_config, execution_path)
```

Comment:
Let's please keep existing tests as-is :)

Reply:
Because of the new implementation, I had no other choice but to add a new parameter to `join_toml_configs()`, and consequently I had to update the tests. What alternative approach would you suggest?

Reply:
Maybe add a default?

Reply:
Fine. :+1: 

Also, to test precedence, I added a parameter to the `pyproject1.toml`: https://github.com/typeddjango/pytest-mypy-plugins/blob/050fb9bc64c98de5456986dcc8714fd3536fb661/pytest_mypy_plugins/tests/test_configs/pyproject1.toml#L7
Consequently, the output of some existing tests changed as well: https://github.com/typeddjango/pytest-mypy-plugins/blob/050fb9bc64c98de5456986dcc8714fd3536fb661/pytest_mypy_plugins/tests/test_configs/test_join_toml_configs.py#L99
Do you want this to be reverted as well?

Reply:
Yes, please. I am asking this, because I want to make sure it won't affect existing use-cases for existing users.